docs/source/tools/assistant.rst

```diff
@@ -106,7 +106,7 @@ Here is what each field means:
 
    The assistant will warn you when it is time to update this setting (see the ``update-check`` setting in the global config  to control how often it checks, or to disable this check entirely).
 - ``name``: the name of the project. This determines the filename of the ``.dar`` file compiled by ``daml build``.
-- ``source``: the location of your main DAML source code file, relative to the project root.
+- ``source``: the location of your DAML source code files.
```

Comment:
I don't think this is very accurate, and I don't see why remove `relative to the project root`?  Perhaps `the location of the project's DAML source code root folder, relative to the project root folder`. But I don't know if there's a better way to say that.

Related: Does `source: .` work as expected? Does `source: .` work on Windows? This could be added as a test. If you omit `source` altogether, it could default to `source: .`

Reply:
improved the sentence, added an entry to release notes.

Reply:
Thank you 🙇‍♀

Reply:
checked for `source: . ` manually on linux, works fine.

Reply:
added it as an integration test.